CSharp/BatchExplorer/readme.txt

```diff
@@ -0,0 +1,11 @@
+﻿Once the app is running, you need to configure it with data from your Azure subscription.
+
+1. On the Accounts menu click "Add Default Account Manager". The "Add Account" dialog opens.
+2. In the Account Alias box, type in any name you like. Use the name to create distinct batch jobs.
```

Comment:
Instead:
"In the Account Alias box, type in any name you like. This is the friendly name of the Batch Account and will show up in the BatchExplorer UI."